.github/workflows/build-standalone.yml

```diff
@@ -28,35 +30,42 @@ jobs:
       - name: Import certificates and setup Keychain
         run: |
           echo "${{ secrets.APPLE_DEVELOPER_ID_CERTIFICATE_BASE64 }}" | base64 --decode > codesign_certificate.p12
+          echo "${{ secrets.APPLE_DEVELOPER_ID_INSTALLER_CERT_BASE64 }}" | base64 --decode > installer_certificate.p12
           echo "${{ secrets.APPLE_APP_STORE_CONNECT_PRIVATE_KEY_BASE64 }}" | base64 --decode > app_store_connect_private_key.p8
 
           security create-keychain -p "$KEYCHAIN_PASSWORD" build.keychain
           security default-keychain -s build.keychain
           security unlock-keychain -p "$KEYCHAIN_PASSWORD" build.keychain
 
           security import codesign_certificate.p12 -k build.keychain -P "${{ secrets.APPLE_DEVELOPER_ID_CERTIFICATE_PASSWORD }}" -T /usr/bin/codesign
+          security import installer_certificate.p12 -k build.keychain -P "${{ secrets.APPLE_DEVELOPER_ID_CERTIFICATE_PASSWORD }}" -T /usr/bin/pkgbuild
           security set-key-partition-list -S apple-tool:,apple: -s -k "$KEYCHAIN_PASSWORD" build.keychain
       - name: Yarn install
         run: yarn install
       - name: Build
         run: yarn build:sea
-      - name: Sign build artifact
+      - name: Codesign binary
         run: |
           codesign --verbose=4 --remove-signature build/sea/p0
-          codesign --verbose=4 --timestamp --entitlements entitlements.plist --options runtime --sign "$APPLE_CODESIGN_CERT_ID" --keychain "build.keychain" build/sea/p0
-      - name: Notarize build Artifact
+          codesign --verbose=4 --timestamp --entitlements entitlements.plist --options runtime --sign "${{ env.APPLE_CODESIGN_CERT_NAME }}" --keychain "build.keychain" build/sea/p0
+      - name: Package application
         run: |
-          zip build/sea/p0-macos-${{ github.event.release.tag_name }}.zip build/sea/p0
-          xcrun notarytool submit build/sea/p0-macos-${{ github.event.release.tag_name }}.zip --key app_store_connect_private_key.p8 --key-id "$APPLE_APP_STORE_CONNECT_KEY_ID" --issuer "$APPLE_APP_STORE_CONNECT_ISSUER_ID" --wait --verbose
+          mkdir -p build/sea/root/usr/local/bin
+          cp build/sea/p0 build/sea/root/usr/local/bin/p0
+          pkgbuild --identifier dev.p0.cli --version "${{ env.VERSION }}" --install-location / --sign "${{ env.APPLE_INSTALLER_CERT_NAME }}" --keychain "build.keychain" --root build/sea/root build/sea/p0-${{ env.VERSION }}.pkg
```

Comment:
This string `dev.p0.cli` needs to be the same between versions. The idiomatic convention is to use a reverse domain-name style string.